oeps/architectural-decisions/oep-0058-arch-translations-management.rst

```diff
@@ -0,0 +1,263 @@
+OEP-58: Translations Management
+###############################
+
+.. list-table::
+   :widths: 25 75
+
+   * - OEP
+     - :doc:`OEP-58 <oep-0058-arch-translations-management>`
+   * - Title
+     - Translations Management
+   * - Last Modified
+     - 2022-11-07
+   * - Authors
+     - 
+       * Carlos Muniz <cmuniz@tcril.org>
+       * Feanil Patel <feanil@tcril.org>
+       * Sarina Canelake <sarina@tcril.org>
+   * - Arbiter
+     - Ned Batchelder <ned@edx.org>
+   * - Status
+     - Under Review
```

Comment:
Based on the [Provisional definition](https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0001.html#provisional) and the [docs for status changes](https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0001.html#status-changes), I'd recommend:
1. Updating the status to `Under Review (=> Provisional)` to clarify the intended status upon merge, assuming that status is agreed upon. And,
2. Adding a `Follow-up Work` page to this header as notes in the docs above. This link might also provide some thoughts on how this work will come to be.

2. Additionally, this isn't a blocker to this OEP, but is related to "Follow-up Work". I'm wondering what thought has been given to how companies will need to handle private repos and translations? Maybe a separate Discuss thread could be started for this, but has thought already been given to this? Is the idea that others could easily replicate this pattern and re-use tooling against a private repo of translations? How much work will that involve? Etc.

Reply:
Thank you for your comments @robrap.

1. If I understand correctly, you want to clarify that the next status after "Under Review" should be "Provisional" and not "Accepted", because it hasn't been vetted and adopted in the platform. This makes sense to me, but I don't think we need to specify this in the document.

2. I will add a link named “Follow-up Work” to the References section of the OEP header.

3. I think this is a good first question for the Follow-up Work link. The openedx-atlas CLI tool can pull translation files from any repository that works with git. And the GitHub Action that generates translations and moves them to openedx-translations can also be reappropriated for private repos. The work involved will be: copying the github action that generates the translations to another repository, changing the organization and names of the repos, and attaching it to a separate Transifex project via the app.

Reply:
@Carlos-Muniz on point 1, @robrap is suggesting that you change the status from:
* `Under Review`

to one of these:

1. `Under Review (=> Provisional)`
2. `Under Review (=> Accepted)`

in order to clarify whether you want this to merge in as a Provisional or Accepted, which is information the reviewers should know. I agree with him.

Reply:
Aside, @robrap : I think this is evidence that the OEP-1 status guidance that we collaborated on could be improved :) OEP-1 presents `Under Review` as a valid status even though we'd really prefer that folks always include the `(=> ...)` part.

Reply:
Thank you for the clarification. I've changed it.